src/lib/components/inputs/AccessLevel.svelte

```diff
@@ -96,13 +105,7 @@
     font-size: var(--font-xs);
     opacity: 0.7;
   }
-  .sr-only {
```

Comment:
Lost the .sr-only class, so now a lone radio input appears when selecting a level, but only when it has focus:
<img width="412" alt="Screenshot 2024-11-20 at 9 43 47 AM" src="https://github.com/user-attachments/assets/cb300b11-6147-4ff0-930c-02ed08cbcb31">